htdocs/xoops_lib/Xoops/Form/Form.php

```diff
@@ -391,16 +402,16 @@ public function setSummary($summary)
             $this->summary = strip_tags($summary);
         }
     }
-
+    
     /**
-     * getExtra - get the extra attributes for the <form> tag
+     * getSummary - get the summary tag for the <form> tag
      *
      * @return string
      */
-    public function getExtra()
+    public function getSummary()
```

Comment:
setSummary() should have been removed, my mistake. It was part of a standard that never got significant traction. Removing it, and new getSummary().